From 4cf1752dc04c17525cca9e45d7651f3aa25af106 Mon Sep 17 00:00:00 2001
From: Xuewei Zhang <xueweiz@google.com>
Date: Tue, 3 Apr 2018 14:50:17 -0700
Subject: [PATCH] konlet script fixes

* Add dependency on gcr-online.target and docker.socket
* Use /usr/share/gce-containers/ as installation path

Signed-off-by: Xuewei Zhang <xueweiz@google.com>
---
 scripts/konlet-startup.service | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/konlet-startup.service b/scripts/konlet-startup.service
index 1fd1cbd..3b228a8 100644
--- a/scripts/konlet-startup.service
+++ b/scripts/konlet-startup.service
@@ -14,11 +14,11 @@
 
 [Unit]
 Description=Containers on GCE Setup
-Requires=network-online.target
-After=network-online.target google-startup-scripts.service
+Wants=network-online.target google-startup-scripts.service gcr-online.target docker.socket
+After=network-online.target google-startup-scripts.service gcr-online.target docker.socket
 
 [Service]
-ExecStart=/usr/bin/konlet-startup
+ExecStart=/usr/share/gce-containers/konlet-startup
 Restart=on-failure
 RestartSec=5
 StandardOutput=journal+console
-- 
2.17.0.484.g0c8726318c-goog

